### PR TITLE
added JS code to cart menu and update html code

### PR DESCRIPTION
**Pull request by salvik21**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,36 +9,41 @@
...
-                    <h3 class="carousel__second-title">for you</h3>
-                </div>
-                <div class="carousel__forward-button"></div>
-                <div class="carousel__back-button"></div>
+        <section class="carousel">
+           <div id="nav__cart-menu">
+              <div id="cart-menu__button"><a target="__blank" href="" >$44.50</a></div>
+              <div id="cart-menu__button"><a target="__blank" href="" >View Cart</a></div>
+              <div id="cart-menu__button"><a target="__blank" href="" >Checkout</a></div>
+           </div>
```
Comment (YuraLeo94) [inline index.html]: It's should be in the nav block.

File: index.html [hunk 2]
```diff
@@ -9,36 +9,41 @@
...
-                    <h1 class="carousel__main-title">made with love</h1>
-                </div>
-                <div class="carousel__second-title-wrapper">
-                    <h3 class="carousel__second-title">for you</h3>
-                </div>
-                <div class="carousel__forward-button"></div>
-                <div class="carousel__back-button"></div>
+        <section class="carousel">
+           <div id="nav__cart-menu">
+              <div id="cart-menu__button"><a target="__blank" href="" >$44.50</a></div>
```
Comment (YuraLeo94) [inline index.html]: When you need assign style more than one element use class.

Comment (YuraLeo94) [reply index.html]: Please change "id" on the "class"

Comment (YuraLeo94) [reply index.html]: Also, why do you need `target="__blank"` here? When you click view cart in an online store, it open a new tab, or it open the cart in the same tab?

File: index.html [hunk 3]
```diff
@@ -9,36 +9,41 @@
...
-                <div class="carousel__main-title-wrapper">
-                    <h1 class="carousel__main-title">made with love</h1>
-                </div>
-                <div class="carousel__second-title-wrapper">
-                    <h3 class="carousel__second-title">for you</h3>
-                </div>
-                <div class="carousel__forward-button"></div>
-                <div class="carousel__back-button"></div>
+        <section class="carousel">
+           <div id="nav__cart-menu">
```
Comment (YuraLeo94) [inline index.html]: think it's better to add the "showMenu_off" class here.

File: java.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const navCart = document.querySelector(".nav__cart");
+const cartMenu = document.getElementById("nav__cart-menu");
+cartMenu.classList.add("showMenu_off");
```
Comment (YuraLeo94) [inline java.js]: you can add it in html code

File: java.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const navCart = document.querySelector(".nav__cart");
```
Comment (YuraLeo94) [inline java.js]: Better use id instead class for this purpose

File: java.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+const cartMenu = document.getElementById("nav__cart-menu");
+cartMenu.classList.add("showMenu_off");
+
+function Cart() {
```
Comment (YuraLeo94) [inline java.js]: I think name `toggleCart` will tell more than just `Cart`

File: java.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+navCart.addEventListener("click", Cart);
+
+function Cart2() {
+  if (cartMenu.style.display === "block") {
+    cartMenu.style.display = "none";
+  } 
+  else {
+    cartMenu.style.display = "block";
+  }
+}
```
Comment (YuraLeo94) [inline java.js]: I not found where you use this function? Do you need this function?

File: style.css [hunk 8]
```diff
@@ -217,4 +217,37 @@ footer {
...
+  background-color: #f5f5f5;
+} 
+
+.showMenu_on{
+  display: block;
+}
+
+.showMenu_off{
+  display: none;
 }
```
Comment (YuraLeo94) [inline style.css]: Better add more specific name `cart-menu--on` `cart-menu--off` or `show-cart-menu--on` `show-cart-menu--off`.

Comment (YuraLeo94) [reply style.css]: Also don't forget about naming with BEM

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]

File: java.js [hunk 9]
```diff
@@ -1,24 +1,14 @@
...
-function Cart() {
-  if(cartMenu.classList.contains("showMenu_on")){
-    cartMenu.classList.remove("showMenu_on");
-    cartMenu.classList.add("showMenu_off");
+function toggleCart() {
+  if(cartMenu.classList.contains("nav__cart-menu--on")){
+    cartMenu.classList.remove("nav__cart-menu--on");
+    cartMenu.classList.add("nav__cart-menu--off");
   }
   else{
```
Comment (YuraLeo94) [inline java.js]: space `else {`

File: java.js [hunk 10]
```diff
@@ -1,24 +1,14 @@
...
+const navCart = document.getElementById("nav__cart");
 const cartMenu = document.getElementById("nav__cart-menu");
-cartMenu.classList.add("showMenu_off");
 
-function Cart() {
-  if(cartMenu.classList.contains("showMenu_on")){
-    cartMenu.classList.remove("showMenu_on");
-    cartMenu.classList.add("showMenu_off");
+function toggleCart() {
+  if(cartMenu.classList.contains("nav__cart-menu--on")){
```
Comment (YuraLeo94) [inline java.js]: also space `if (cartMenu.classList.contains("nav__cart-menu--on")) {`

File: java.js (hunk 9 above)
Comment (YuraLeo94) [reply java.js]: You can try the next key combination `alt+shift+f`, hope it will help you do auto-correction for this file.

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
Sorry, forgot to write before. A better name for a file with `.js` extension will be what is inside this file. In the current case, it can name like `cart.js`, since you do functionality for cart, not `java`. **Java** is a _programming language_.
Also when you will start to implement functionality for the menu and carousel you can create two files `menu.js` and `carousel.js` and etc.

File: java.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+const navCart = document.getElementById("nav__cart");
+const cartMenu = document.getElementById("nav__cart-menu");
+
+function toggleCart() {
```
Comment (CoobQ) [inline java.js]: Your function is called toggle but it doesn't toggle. please check https://www.w3schools.com/howto/howto_js_toggle_class.asp

**Review by YuraLeo94 — APPROVED** [review]

**Pull request merged**